spotify/models/user.py

```diff
@@ -450,6 +450,20 @@ def refresh(self):
         playlist_data = await self.http.create_playlist(self.id, **data)
         return Playlist(self.__client, playlist_data, http=self.http)
 
+    @ensure_http
+    async def follow_playlist(self, playlist, public=True):
```

Comment:
Missing type annotations

Reply:
I'll add the annotation playlist: Union[str, Playlist].

Do you want me to add the  ,*, in the args?  I'm guessing that is used to force people to use kwd args?

Reply:
Yeah the `, *,` pattern is used to consume all remaining positional arguments and discard them, any arguments after this will be keyword only arguments and if you do not specify defaults for the kwonly args they must be passed in explicitly named.

I wouldn't be against forcing the `public` argument to be keyword only